### DISCUSSION TITLE
Fullstack Extension: Transfer Ownership / Unpublish?

### DISCUSSION THREAD

**Discussion by karlhorky** [streetsidesoftware/cspell-dicts#271] [Archive]
Hi there!

First of all, thanks for your work on this extension and the associated dictionaries - really cool!

I was looking at the Fullstack dictionary extension on the VS Code marketplace here, which is currently owned by @rbalet:

https://marketplace.visualstudio.com/items?itemName=rbalet.code-spell-checker-fullstack

It has a number of problems (old publish date, points at old repository, etc):

<img src=https://user-images.githubusercontent.com/1935696/98532430-f7e46c80-2281-11eb-9f08-8f7de18b50dd.png width=300>

Is this dictionary now included in the main extension? If so, maybe the Fullstack dictionary extension should be unpublished?

@rbalet, would you be willing to help with this?

**Comment by Jason3S** [comment]
@karlhorky thank you for bringing this up. I don't have any control over it.

**Comment by karlhorky** [comment]
Ok, I have reached out to Raphaël over email also separately from this issue, let's see!

**Edit:** Raphaël is trying to unpublish the package now.

**Reply by rbalet** [reply]
Hi @karlhorky.

I was finally able to unpublish the extension so you may close the issue.

**Reply by karlhorky** [reply]
Great, thanks @rbalet ! I think it's a GitHub Discussion now, so I don't think I can close it.

@Jason3S is there a way to lock or close this?
